workshops/sprig_first_game/README.md

```diff
@@ -7,9 +7,9 @@ img: "https://user-images.githubusercontent.com/27078897/186008909-cc9ea9d5-5843
 
 Let's make our first game in Sprig!
 
-[Sprig](https://editor.sprig.hackclub.com/) is a tool developed by Hack Club to help people quickly get started making interesting tile games. It's a tiny game engine embedded in a web-based editor with integrated art and music tools. If you build a game in Sprig and share it in the community gallery you can get a [Sprig Console](https://sprig.hackclub.com)!
+[Sprig](https://sprig.hackclub.com/) is a tool developed by Hack Club to help people quickly get started making interesting tile games. It's a tiny game engine embedded in a web-based editor with integrated art and music tools. If you build a game in Sprig and share it in the community gallery you can get a [Sprig Console](https://sprig.hackclub.com)!
 
-We're going to build a Sokoban game to get started with Sprig. Let's [open up Sprig](https://sprig.hackclub.com) first:
+We're going to build a Sokoban game to get started with Sprig. Find the Sokoban game [in the gallery] (https://sprig.hackclub.com/gallery) and click on it to open the editor. Now you're hacking on Sokoban's code! You can do this with any game in the gallery.
```

Comment:
I don't think the intention of this workshop is to start with Sokoban, I think it's to build Sokoban

Reply:
I'm glad you caught that